docs/cody/capabilities/ignore-context.mdx

```diff
@@ -1,22 +1,157 @@
+
+When making requests to third-party LLMs, Cody can use content from a repository if its name matches any of the `include` patterns and does not match any of the `exclude` patterns. Thus, the `exclude` rules filter only the repositories allowed by the `include` rules. For example,
+
+```
+{
+ 'cody.contextFilters': {
+ // All repositories starting with "github.com/sourcegraph/" are allowed
+ include: [{repoNamePattern: '^github\\.com\\/sourcegraph\\/.+'}],
+ // except for "github.com/sourcegraph/cody-analytics"
+ // and the ones containing "cody" in their name.
+ exclude: [
+ {repoNamePattern: '^github\\.com\\/sourcegraph\\/cody-analytics$'},
+ {repoNamePattern: '.*cody.*'}
+ ]
+ }
+}
+```
+
+In this case, Cody can access content from repositories whose names start with `github.com/sourcegraph/`. However, it excludes files from repositories with `cody` in their name.
+
+## Context Filters Compatibility Matrix
+
+<Callout type="info"> Cody Context Filters work only with Sourcegraph versions >=5.4.0 for Enterprise users.</Callout>
+
+Depending on the client type, here's a detailed breakdown of all the prerequisites that are supported for the context filters to work.
+
+|        **Client Versions**        | Sourcegraph `v<5.4.0` | Sourcegraph `v=5.4.0`                           | Sourcegraph `v=5.4.X`                           | Sourcegraph `v>=5.4.0` |
+| --------------------------------- | --------------------- | ------------------------------------------------------------------------ | ------------------------------------------------------------------------- | ---------------------- |
+| **JetBrains `<= 5.5.9`**          | N/A          | Respects policy, but the server returns an unhelpful error | Respects policy, the server returns compatibility check and prompt to upgrade | Not affected           |
+| **`5.5.10 <= JetBrains < 6.0.0`** | N/A          | Respects policy, but the server only returns a compatibility check           | Respects policy, the server returns compatibility check and prompts to upgrade | Not affected           |
+| **JetBrains `>= 6.0.0`**          | N/A          | Respects policy                                                          | Respects policy                                                           | Not affected           |
+| **VS Code `< 1.16.X`**            | N/A          | Respects policy, but the server returns an unhelpful error | Respects policy, the server returns compatibility check and prompt to upgrade | Not affected           |
+| **`1.18.0 < VS Code < 1.20.0`**   | N/A          | Respects policy, but the server only returns a compatibility check           | Respects policy, the server returns compatibility check and prompt to upgrade | Not affected           |
+| **VS Code `>= 1.20.0`**           | N/A          | Respect policy                                                           | Respects policy                                                           | Not affected           |
```

Comment:
```suggestion
| **JetBrains `<= 5.5.9`**          | N/A          | Cody is not compatible | Respects policy and prompts to upgrade | Not affected           |
| **`5.5.10 <= JetBrains < 6.0.0`** | N/A          | Cody is not compatible | Respects policy and prompts to upgrade | Not affected           |
| **JetBrains `>= 6.0.0`**          | N/A          | Respects policy | Respects policy | Not affected |
| **VS Code `< 1.16.X`**            | N/A          | Cody is not compatible | Respects policy and prompts to upgrade | Not affected           |
| **`1.18.0 < VS Code < 1.20.0`**   | N/A          | Cody is not compatible | Respects policy and prompts to upgrade | Not affected           |
| **VS Code `>= 1.20.0`**           | N/A          | Respects policy  | Respects policy   | Not affected           |
```

Reply:
@MaedahBatool @aramaraju I simplified the wording, I don't think we need to tell customers what gets returned when there is an incompatible version. What do you think?

Reply:
You were a few mins ahead of me. My thoughts are in my comment [here](https://github.com/sourcegraph/docs/pull/326#discussion_r1617938802)

Reply:
Thank you :)